src/CSET/operators/plot.py

```diff
@@ -96,17 +96,18 @@ def spatial_contour_plot(
 
 
 def postage_stamp_contour_plot(
-    cube: iris.cube.Cube, file_path: Path, **kwargs
+    cube: iris.cube.Cube, file_path: Path, coordinate: str = "realization", **kwargs
 ) -> iris.cube.Cube:
-    """
-    Plots postage stamp contour plots from an ensemble.
+    """Plots postage stamp contour plots from an ensemble.
 
     Parameters
     ----------
     cube: Cube
         Iris cube of data to be plotted. It must have a realization coordinate.
     file_path: pathlike
         The path of the plot to write.
+    coordinate: str
+        The coordinate that becomes different plots. Defaults to "realization".
```

Comment:
I guess this could be also informed by the stratify operator so does not necessarily need to be realisation

Reply:
We could have it with no default, so it has to be specified, but I consider realization to be the common case for postage stamp plots.